### DISCUSSION TITLE
Blocky pages

### DISCUSSION THREAD

**Discussion by aliulka** [stephanrauh/ngx-extended-pdf-viewer#2093] [Q&A]
Hi everyone, I have one issue.
![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/135831632/fa9e4644-8f09-46a3-9cd5-c7b606dfd9c7)
By some reason sometimes general page image and thumbnails looks like at the screen above. There should be tables with data instead. If I'll zoom page it will turns into normal view with correct tables and data. Please help me to figure out how to make pages and thumbnails normal)

**Comment by vzahriichuk** [comment]
@stephanrauh Hi!
Do you have any idea why this could happen and how we can resolve it?

**Reply by stephanrauh** [reply]
No. I'm clueless.

BTW, sorry for answering late - I was convinced I already had answered you guys, but obviously I didn't.
